src/raven.js

```diff
@@ -413,6 +413,16 @@ Raven.prototype = {
             timestamp: now() / 1000
         }, obj);
 
+        if (isFunction(this._globalOptions.breadcrumbCallback)) {
+            var result = this._globalOptions.breadcrumbCallback(crumb);
+
+            if (result) {
+                crumb = result;
+            } else {
```

Comment:
One other note since we brought `null` into the picture - since `typeof null` is `object`, we should be careful to do the "is it false/null" case first, so that in the second case when you do something like `if typeof result === 'object'`, we won't mistakenly treat `null` as an object.